src/scripts/validation.js

```diff
@@ -7,14 +7,6 @@ const settings = {
   errorClass: "modal__error_visible",
 };
 
-const enableValidation = (config) => {
-  const formList = Array.from(document.querySelectorAll(config.formSelector));
-
-  formList.forEach((formEl) => {
-    setEventListeners(formEl, config);
-  });
-};
-
 function disableButton(buttonElement, inactiveButtonClass) {
```

Comment:
Please, export `disableButton` from `validation.js` to be able to use it in `index.js`